integration-tests/gradle/src/test/java/io/quarkus/gradle/ListExtensionsTest.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.gradle;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class ListExtensionsTest extends QuarkusGradleWrapperTestBase {
+
+    @Test
+    public void testListExtensionsWork() throws IOException, URISyntaxException, InterruptedException {
+
+        final File projectDir = getProjectDir("list-extension-single-module");
+        runGradleWrapper(projectDir, ":listExtension");
+
+        List<String> outputLogLines = listExtensions(projectDir, ":listExtension");
+
+        assertThat(outputLogLines).anyMatch(line -> line.contains("quarkus-resteasy"));
```

Comment:
BTW I think there is already an assertion in AssertJ that asserts if a line exists in a given File `assertThat(file).hasContent(...)`,  but this okay

Reply:
Indeed, but I think we're fine with this for now :)